src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesPairs = sourceString.split(';');
+  const stylesArray = stylesPairs.map((pair) => {
+    return pair
+      .trim()
+      .split(':')
+      .map((part) => part.trim());
+  });
+
+  const stylesObject = {};
+
+  stylesArray.forEach(([key, value]) => {
+    stylesObject[key] = value;
```

Comment:
[CODE STYLE] - The variable naming `stylesObject` could be improved to reflect what it represents more clearly. Consider a name that indicates the purpose or content of the object, such as `styleRules` or `cssProperties`, to enhance code readability.